presto-main/src/main/antlr4/com/facebook/presto/type/TypeCalculation.g4

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+grammar TypeCalculation;
+
+typeCalculation
+    : expression
+    ;
+
+expression
+    : NULL                                                                    #nullLiteral
+    | INTEGER_VALUE                                                           #numericLiteral
+    | IDENTIFIER                                                              #identifier
+    | '(' expression ')'                                                      #parenthesizedExpression
```

Comment:
see my commit https://github.com/sopel39/presto/commit/bdc3a5ffdbd0e4d30091e9b071a8e8ef55984d33 how this can be fixed